base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/controls/AlarmButtonViewModel.java

```diff
@@ -33,20 +33,18 @@
     public AlarmButtonViewModel(final AlarmCounter alarmCounter, String buttonLabel) {
         super(buttonLabel);
 
-        flash = new FlashingButton(this);
+        flash = new FlashingButton(this, ALARM_COLOR);
         flash.setDefaultColour(DEFOCUSSED);
 
-        alarmCounter.addPropertyChangeListener("alarmCount", new PropertyChangeListener() {
-            @Override
-            public void propertyChange(PropertyChangeEvent arg0) {
+        alarmCounter.addPropertyChangeListener("alarmCount", arg0 -> {
                 alarmCount = alarmCounter.getCount();
                 if (!active) {
                     alarmSeen = false;
                 }
                 update();
                 updateFlashing();
             }
-        });
+        );
```

Comment:
Confusing indentation, did you mean to delete the other bracket?

Reply:
Fixed.